src/makeRobotAccountant.js

```diff
@@ -6,7 +6,21 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let counter = 0;
+
+  const getSum = function(firstNumber) {
+    return (secoundNumber) => {
+      counter++;
+
+      if (counter <= 3 || counter % 2 !== 0) {
+        return firstNumber + secoundNumber;
+      }
+
+      return 'Bzzz... Error!';
+    };
+  };
+
+  return getSum;
```

Comment:
```suggestion
  return function (firstNumber) {
    return (secoundNumber) => {
      counter++;

      if (counter <= 3 || counter % 2 !== 0) {
        return firstNumber + secoundNumber;
      }

      return 'Bzzz... Error!';
    };
  };
```

Reply:
https://github.com/mate-academy/js_advanced_calculator/pull/1496#pullrequestreview-1422366754

Okay, there is a point to that, but here's another task where it was necessary to declare an object for readability and then return it for better readability.
Moreover, in my opinion, we improve code readability by getting rid of such nesting.

    };
  };
}